app/models/sample.rb

```diff
@@ -166,6 +166,10 @@ def filter_host_flag
     host_genome && host_genome.name == HostGenome::NO_HOST_NAME ? 0 : 1
   end
 
+  def skip_deutero_filter_flag
+    !host_genome || host_genome.skip_deutero_filter == 1 ? 1 : 0
```

Comment:
Did you mean to skip deutero filtering if there is no host genome?

Reply:
Yes: deutero filtering is really a part of host subtraction